File: oldPro/WZCusPro/app/src/main/java/com/wzcuspro/app/base/BaseActivity.java
package com.wzcuspro.app.base;

import android.os.Bundle;
import android.support.annotation.Nullable;

public abstract class BaseActivity extends BaseSwipActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        try {
            int resId = layoutResId();
            if (resId != 0) {
                setContentView(resId);
            } else {
                setContentView(createView());
            }
            initView();
            initData();
            styleBar(this);
        } catch (Exception e) {
            System.out.print("捕获初始化异常===>>>:" + e.getMessage());
        }
    }
}

File: KtxPro/KTX2/app/src/main/java/com/ktx2/base/BaseActivity.kt
package com.ktx2.base

import android.app.Activity
import android.os.Bundle

abstract class BaseActivity : BaseInitActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        try {
            setContentView(layoutId())
            initView()
            styleStatus(this)
            initData()
        } catch (e: Exception) {
            println("base init error ${e.message}")
        }

    }

    fun styleStatus(activity: Activity) {}




}